manager/integration/tests/test_basic.py

```diff
@@ -4138,9 +4140,14 @@ def test_volume_toomanysnapshots_condition(client, core_api, volume_name):  # NO
     volume.snapshotDelete(name=snap[101].name)
     volume.snapshotDelete(name=snap[100].name)
 
-    volume.snapshotPurge()
-    volume = wait_for_snapshot_purge(client, volume_name,
-                                     snap[101].name, snap[100].name)
+    # TODO: engine doesn't have purge status for v2 data engine
+    # It will be implemented in
+    # https://github.com/longhorn/longhorn/issues/11833
+    enable_v2 = os.environ.get('RUN_V2_TEST')
```

Comment:
Could we use `DATA_ENGINE` here to align with other test cases? beside this no other question, thank you.